Introduce composer docs:generate (#290)
TYPO3-Documentation/TYPO3CMS-Reference-TSconfig: CONTRIBUTING.md

# Contribute to TYPO3 Documentation

## Render documentation locally

Render documentation locally:

```
composer install --ignore-platform-reqs
composer docs:generate
```

## Create Issues

> * If you find something missing or something is wrong in this manual, you are welcome to write an issue describing the problem.

Reviewer: ```suggestion
* If you find something missing or something is wrong in this manual, you are welcome to open an issue and describe the problem.
```

> * If you can, please try to fix the problem yourself.

Reviewer: ```suggestion
* Please feel free to submit a fix to the problem.
```
